scripts/smoothwall/HandleDansGuardianLog.pl

```diff
@@ -1,201 +0,0 @@
-#!/usr/bin/perl
```

Comment:
Just a quick one: why is this file being deleted?

Reply:
Moved to modules/yona/bin
(basically, laying it out into into a Smoothwall format)

Reply:
Git did not pick it up as a move.  I didn't bother trying to figure out why.

Reply:
I guess you still need to bother, as the "new" Perl script is not part of this PR, so now it comes as a delete, while you were expecting a delete/create and where I hoping for a move, so we retain the history.

Reply:
Files were missing due to the project gitignore for general 'bin' exclude.